packages/plugins/Routing/src/utils/routingServiceUtils.ts

```diff
@@ -0,0 +1,54 @@
+import { transform } from 'ol/proj'
+
+async function fetchRoutingDirections(
+  url: string,
+  searchCoordinates: number[][],
+  selectedRouteTypesToAvoid: string[],
+  selectedPreference: string,
+  apiKey: string
+) {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      /* eslint-disable @typescript-eslint/naming-convention */
+      'Content-Type': 'application/json',
+      Authorization: apiKey,
+      /* eslint-enable @typescript-eslint/naming-convention */
+    },
+    body: JSON.stringify({
+      coordinates: searchCoordinates,
+      geometry: true,
+      instructions: true,
+      language: 'en',
```

Comment:
It would be nice to use the user's language here as well.
```suggestion
      language: i18next.language,
```